list/styles2-lists.css

```diff
@@ -0,0 +1,78 @@
+/* Lists Main */
+.lists-container {
+  margin: 3rem;
+}
+
+.lists-main-heading {
+  color: yellow;
+}
+.lists-snippet iframe {
+  margin: 1rem 0rem;
+  width: 524px;
+  height: 200px;
+  border: 0;
+  transform: scale(1);
+  overflow: hidden;
+}
+.lists-sub {
+  margin: 1rem 0rem;
+  color: white;
+  font-size: 1.2rem;
+}
+.lists-snippet {
+  margin: 1rem 0rem;
+}
+.standard-lists {
+  color: yellow;
+  margin: 1rem 0rem;
+}
+
+.spaced-list {
+  background-image: linear-gradient(to right, #2adeeb, #a517ba);
+  color: white;
+  font-size: 1.3rem;
+  text-align: center;
+}
+
+.list-inline {
+  list-style: none;
+}
+
+.list-inline-item {
+  display: inline-block;
+  padding: 1rem;
+}
+.list-inline-item:hover {
+  transform: scale(1.1);
+  font-weight: bold;
```

Comment:
you can use multiple of 4 or 8 in case of font-weight, it is a professional convention

Reply:
oky